// @scout/tokens/dist/ts/tokens.js
/**
 * Do not edit directly, this file was auto-generated.
 */

export const ScoutBorderWidth0 = "0px";
export const ScoutBorderWidth1 = "1px";
export const ScoutBorderWidth2 = "2px";
export const ScoutBorderNone = "#00000000";
export const ScoutBorderPrimary = "#d1d5db";
export const ScoutBorderSecondary = "#e5e7eb";
export const ScoutBorderDisabled = "#e5e7eb"; // Disabled inputs and controls.
export const ScoutBorderInverse = "#ffffff33"; // Border drawn on top of inverse / dark surfaces (snackbar, tile splits).
export const ScoutBorderInfo = "#029dd4";
export const ScoutBorderWarning = "#eab308";
export const ScoutBorderCritical = "#cc2427"; // Renamed from `error` for consistency with fill.critical and the component status vocabulary.
export const ScoutBorderSuccess = "#47b52c";
export const ScoutBorderTeal = "#4f8aa0"; // Teal border paired with `fill.teal` — used by share-with-customer surfaces.
export const ScoutColorRed50 = "#fdf3f3";
export const ScoutColorRed100 = "#fbe9e9";
export const ScoutColorRed200 = "#f7caca";
export const ScoutColorRed300 = "#ef9595";
export const ScoutColorRed400 = "#e05252";
export const ScoutColorRed500 = "#cc2427";
export const ScoutColorRed600 = "#a71b20";
export const ScoutColorRed700 = "#85141a";
export const ScoutColorRed800 = "#640c12";
export const ScoutColorRed900 = "#3f0710";
export const ScoutColorRed950 = "#250308";
export const ScoutColorRed999 = "#150103";
export const ScoutColorYellow50 = "#fefce8";
export const ScoutColorYellow100 = "#fefbd9";
export const ScoutColorYellow200 = "#fef08a";
export const ScoutColorYellow300 = "#fde047";
export const ScoutColorYellow400 = "#facc15";
export const ScoutColorYellow500 = "#eab308";
export const ScoutColorYellow600 = "#ca8a04";
export const ScoutColorYellow700 = "#a16207";
export const ScoutColorYellow800 = "#854d0e";
export const ScoutColorYellow900 = "#713f12";
export const ScoutColorYellow950 = "#422006";
export const ScoutColorYellow999 = "#1f0e02";
export const ScoutColorGreen50 = "#f4fcf0";
export const ScoutColorGreen100 = "#eef9eb";
export const ScoutColorGreen200 = "#d6f2cf";
export const ScoutColorGreen300 = "#ace69d";
export const ScoutColorGreen400 = "#73d85a";
export const ScoutColorGreen500 = "#47b52c";
export const ScoutColorGreen600 = "#359220";
export const ScoutColorGreen700 = "#277217";
export const ScoutColorGreen800 = "#17520f";
export const ScoutColorGreen900 = "#0e3409";
export const ScoutColorGreen950 = "#072105";
export const ScoutColorGreen999 = "#031003";
export const ScoutColorTeal50 = "#f6fafb";
export const ScoutColorTeal100 = "#edf5f7";
export const ScoutColorTeal200 = "#d5e6ec";
export const ScoutColorTeal300 = "#afceda";
export const ScoutColorTeal400 = "#80b0c2";
export const ScoutColorTeal500 = "#4f8aa0";
export const ScoutColorTeal600 = "#3c6d80";
export const ScoutColorTeal700 = "#2c5263";
export const ScoutColorTeal800 = "#1e3b48";
export const ScoutColorTeal900 = "#162a33";
export const ScoutColorTeal950 = "#0c1a21";
export const ScoutColorTeal999 = "#060f13";
export const ScoutColorBlue50 = "#eefcff";
export const ScoutColorBlue100 = "#e7f8fe";
export const ScoutColorBlue200 = "#c4eefd";
export const ScoutColorBlue300 = "#88dcfb";
export const ScoutColorBlue400 = "#37c7fb";
export const ScoutColorBlue500 = "#029dd4";
export const ScoutColorBlue600 = "#047eaf";
export const ScoutColorBlue700 = "#066289";
export const ScoutColorBlue800 = "#064360";
export const ScoutColorBlue900 = "#02314a";
export const ScoutColorBlue950 = "#011d2e";
export const ScoutColorBlue999 = "#000d17";
export const ScoutColorPurple50 = "#faf5ff";
export const ScoutColorPurple100 = "#f3e8ff";
export const ScoutColorPurple200 = "#e9d5ff";
export const ScoutColorPurple300 = "#d8b4fe";
export const ScoutColorPurple400 = "#c084fc";
export const ScoutColorPurple500 = "#a855f7";
export const ScoutColorPurple600 = "#9333ea";
export const ScoutColorPurple700 = "#7e22ce";
export const ScoutColorPurple800 = "#6b21a8";
export const ScoutColorPurple900 = "#581c87";
export const ScoutColorPurple950 = "#3b0764";
export const ScoutColorPurple999 = "#1d032d";
export const ScoutColorCoolGray50 = "#f9fafb";
export const ScoutColorCoolGray100 = "#f3f4f6";
export const ScoutColorCoolGray200 = "#e5e7eb";
export const ScoutColorCoolGray300 = "#d1d5db";
export const ScoutColorCoolGray400 = "#9ca3af";
export const ScoutColorCoolGray500 = "#6b7280";
export const ScoutColorCoolGray600 = "#4b5563";
export const ScoutColorCoolGray700 = "#374151";
export const ScoutColorCoolGray800 = "#1f2937";
export const ScoutColorCoolGray900 = "#111827";
export const ScoutColorCoolGray950 = "#030712";
export const ScoutColorCoolGray999 = "#000204";
export const ScoutColorWarmGray50 = "#fafaf9";
export const ScoutColorWarmGray100 = "#f5f5f4";
export const ScoutColorWarmGray200 = "#e7e5e4";
export const ScoutColorWarmGray300 = "#d6d3d1";
export const ScoutColorWarmGray400 = "#a8a29e";
export const ScoutColorWarmGray500 = "#78716c";
export const ScoutColorWarmGray600 = "#57534e";
export const ScoutColorWarmGray700 = "#44403c";
export const ScoutColorWarmGray800 = "#292524";
export const ScoutColorWarmGray900 = "#1c1917";
export const ScoutColorWarmGray950 = "#0c0a09";
export const ScoutColorWarmGray999 = "#040302";
export const ScoutColorAlpha10 = "#0000001a";
export const ScoutColorAlpha20 = "#00000033";
export const ScoutColorAlpha80 = "#000000cc";
export const ScoutColorAlphaWhite10 = "#ffffff1a";
export const ScoutColorAlphaWhite20 = "#ffffff33";
export const ScoutColorAlphaWhite80 = "#ffffffcc";
export const ScoutColorWhite = "#ffffff";
export const ScoutColorBlack = "#000000";
export const ScoutElevation1 =
  "0 1px 2px rgba(0,0,0,0.06), 0 1px 1px rgba(0,0,0,0.04)";
export const ScoutElevation2 =
  "0 2px 4px rgba(0,0,0,0.08), 0 1px 2px rgba(0,0,0,0.04)";
export const ScoutElevation3 =
  "0 8px 16px rgba(0,0,0,0.10), 0 2px 4px rgba(0,0,0,0.06)";
export const ScoutElevation4 =
  "0 16px 32px rgba(0,0,0,0.14), 0 4px 8px rgba(0,0,0,0.08)";
export const ScoutMotionDurationInstant = "0ms"; // Reduced-motion fallback.
export const ScoutMotionDurationHover = "180ms"; // Subtle, on-the-edge-of-perception pace for hover and focus state transitions. Slow enough to feel intentional, fast enough to feel responsive.
export const ScoutMotionDurationFast = "150ms"; // Small-scale transitions (toasts dismissing, tooltips appearing).
export const ScoutMotionDurationBase = "250ms"; // Default — toasts, tooltips, most enter/exit.
export const ScoutMotionDurationSlow = "400ms"; // Modal, drawer, emphasis.
export const ScoutMotionDurationDeliberate = "600ms"; // Page-level transitions.
export const ScoutMotionEasingLinear = "linear";
export const ScoutMotionEasingStandard = "cubic-bezier(0.2, 0, 0, 1)"; // Default UI motion.
export const ScoutMotionEasingGentle = "cubic-bezier(0.4, 0, 0.2, 1)"; // Softer ease-out for hover / focus changes — eases in and out symmetrically so the change feels like settling, not snapping.
export const ScoutMotionEasingEnter = "cubic-bezier(0, 0, 0, 1)"; // Decelerate-in.
export const ScoutMotionEasingExit = "cubic-bezier(0.4, 0, 1, 1)"; // Accelerate-out.
export const ScoutMotionEasingEmphasis = "cubic-bezier(0.2, 0, 0, 1.4)"; // Slight overshoot.
export const ScoutRadius0 = "0px";
export const ScoutRadius2 = "2px";
export const ScoutRadius4 = "4px";
export const ScoutRadius8 = "8px";
export const ScoutRadius12 = "12px";
export const ScoutRadius999 = "9999px";
export const ScoutSpace0 = "0px";
export const ScoutSpace4 = "4px";
export const ScoutSpace8 = "8px";
export const ScoutSpace12 = "12px";
export const ScoutSpace16 = "16px";
export const ScoutSpace24 = "24px";
export const ScoutSpace32 = "32px";
export const ScoutSpace48 = "48px";
export const ScoutSpace64 = "64px";
export const ScoutSpace96 = "96px";
export const ScoutStrokeSolid = "solid";
export const ScoutStrokeDashed = "dashed";
export const ScoutFontFamilyLiterata =
  "Literata, Georgia, 'Times New Roman', serif";
export const ScoutFontFamilyInter =
  "Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif";
export const ScoutFontWeightRegular = 400;
export const ScoutFontWeightSemibold = 600;
export const ScoutFontSize10 = "10px";
export const ScoutFontSize12 = "12px";
export const ScoutFontSize14 = "14px";
export const ScoutFontSize16 = "16px";
export const ScoutFontSize20 = "20px";
export const ScoutFontSize24 = "24px";
export const ScoutFontSize32 = "32px";
export const ScoutFontSize48 = "48px";
export const ScoutFontSize64 = "64px";
export const ScoutFontLineHeight15 = "15px";
export const ScoutFontLineHeight18 = "18px";
export const ScoutFontLineHeight21 = "21px";
export const ScoutFontLineHeight24 = "24px";
export const ScoutFontLineHeight30 = "30px";
export const ScoutFontLineHeight36 = "36px";
export const ScoutFontLineHeight40 = "40px";
export const ScoutFontLineHeight64 = "64px";
export const ScoutFontLineHeight76 = "76px";
export const ScoutFontLetterSpacing0 = "0";
export const ScoutZIndexBase = 0;
export const ScoutZIndexRaised = 10;
export const ScoutZIndexSticky = 100;
export const ScoutZIndexOverlay = 1000;
export const ScoutZIndexModal = 2000;
export const ScoutZIndexContextual = 5000;
export const ScoutZIndexDynamic = 9000;
export const ScoutFillAlwaysWhite = "#ffffff"; // Plain white fill that does NOT theme — stays white in dark mode. Used by the tile family.
export const ScoutFillPrimary = "#eefcff"; // Subtle brand-tinted fill (alias of `fill.info.subtle`).
export const ScoutFillSecondary = "#f3f4f6"; // Subtle neutral fill.
export const ScoutFillInfoSubtle = "#e7f8fe";
export const ScoutFillInfoBold = "#029dd4";
export const ScoutFillSuccessSubtle = "#eef9eb";
export const ScoutFillSuccessBold = "#47b52c";
export const ScoutFillWarningSubtle = "#fefbd9";
export const ScoutFillWarningBold = "#eab308";
export const ScoutFillCriticalSubtle = "#fbe9e9";
export const ScoutFillCriticalBold = "#cc2427";
export const ScoutFillAiSubtle = "#f3e8ff";
export const ScoutFillAiBold = "#a855f7";
export const ScoutFillTeal = "#edf5f7"; // Teal-tinted fill — used by share-with-customer surfaces and other agent-only callouts that should feel distinct from the info/blue family.
export const ScoutFocusRingColor = "#029dd4";
export const ScoutFocusRingWidth = "2px";
export const ScoutFocusRingOffset = "2px";
export const ScoutFocusRingCriticalColor = "#cc2427"; // Focus ring on invalid form fields.
export const ScoutIconDisplayPrimary = "#1f2937";
export const ScoutIconDisplaySecondary = "#4b5563";
export const ScoutIconDisplayDisabled = "#9ca3af";
export const ScoutIconDisplayInfo = "#047eaf";
export const ScoutIconDisplayWarning = "#ca8a04";
export const ScoutIconDisplayCritical = "#a71b20";
export const ScoutIconDisplaySuccess = "#359220";
export const ScoutIconDisplayIncrease = "#359220";
export const ScoutIconDisplayDecrease = "#a71b20";
export const ScoutIconInteractivePrimary = "#1f2937";
export const ScoutIconInteractiveSecondary = "#4b5563";
export const ScoutIconInteractiveDelete = "#cc2427";
export const ScoutIconInteractiveInfo = "#047eaf";
export const ScoutIconInteractiveWarning = "#ca8a04";
export const ScoutIconInteractiveCritical = "#a71b20";
export const ScoutIconInteractiveSuccess = "#359220";
export const ScoutIconInteractiveIncrease = "#359220";
export const ScoutIconInteractiveDecrease = "#a71b20";
export const ScoutIconInversePrimary = "#ffffff";
export const ScoutIconInverseSecondary = "#ffffffcc";
export const ScoutIllustrationPrimary = "#029dd4"; // Main brand color in the illustration (figure outline, primary stroke).
export const ScoutIllustrationSecondary = "#d1d5db"; // Neutral support color (background shapes, dividers).
export const ScoutIllustrationAccent = "#e7f8fe"; // Soft brand-tinted highlight (gradient stops, subtle fills).
export const ScoutIllustrationSurface = "#f3f4f6"; // Background surface for the illustration scene.
export const ScoutInteractiveBackgroundHover = "#f3f4f6"; // Default hover background for buttons, list items, dropdown options.
export const ScoutInteractiveBackgroundPressed = "#e5e7eb"; // Default active / pressed background.
export const ScoutInteractiveBackgroundSelected = "#f3f4f6"; // Resting selected background (segmented control track, sidebar selected row).
export const ScoutInteractiveBackgroundHoverOnTint = "#0000001a"; // Subtle black-alpha hover for controls sitting on a tinted surface (snackbar / inline-alert dismiss). The neutral hover (cool-gray.100) disappears against status fills, so this overlay reads on any tint and on both themes.
export const ScoutInteractiveBackgroundPressedOnTint = "#00000033"; // Pressed companion to `hover-on-tint`.
export const ScoutInteractiveBackgroundBrandHover = "#eefcff"; // Hover for brand-tinted neutral surfaces (anchor-links rest hover, sidebar nav rest hover).
export const ScoutInteractiveBackgroundBrandPressed = "#e7f8fe"; // Pressed for brand-tinted neutral surfaces.
export const ScoutInteractiveBackgroundBrandStrong = "#029dd4"; // Filled brand surface — primary button rest.
export const ScoutInteractiveBackgroundBrandStrongHover = "#047eaf"; // Filled brand surface hovered.
export const ScoutInteractiveBackgroundBrandStrongPressed = "#066289"; // Filled brand surface pressed.
export const ScoutInteractiveBackgroundCriticalHover = "#fdf3f3";
export const ScoutInteractiveBackgroundCriticalPressed = "#fbe9e9";
export const ScoutInteractiveBackgroundCriticalStrong = "#cc2427"; // Filled critical surface — critical button rest.
export const ScoutInteractiveBackgroundCriticalStrongHover = "#a71b20";
export const ScoutInteractiveBackgroundCriticalStrongPressed = "#85141a";
export const ScoutInteractiveBackgroundSuccessStrong = "#359220"; // Filled success / affirmative surface — action button rest.
export const ScoutInteractiveBackgroundSuccessStrongHover = "#277217";
export const ScoutInteractiveBackgroundSuccessStrongPressed = "#17520f";
export const ScoutSurfacePage = "#f3f4f6"; // Document background.
export const ScoutSurfacePrimary = "#ffffff"; // Default raised surface (cards, dialogs, popovers).
export const ScoutSurfaceScrim = "#000000cc"; // Modal / drawer backdrop.
export const ScoutSurfaceDisabled = "#f9fafb"; // Background for disabled inputs and controls.
export const ScoutSurfaceInverse = "#1f2937"; // Dark surface — snackbar, tooltip, dark-mode tile splits.
export const ScoutSurfaceInverseStrong = "#111827"; // Slightly darker inverse for layered dark surfaces.
export const ScoutTextDisplayPrimary = "#1f2937";
export const ScoutTextDisplaySecondary = "#4b5563";
export const ScoutTextDisplayDisabled = "#9ca3af"; // Disabled inputs, dimmed labels, locked controls.
export const ScoutTextDisplayInfo = "#047eaf";
export const ScoutTextDisplayWarning = "#ca8a04";
export const ScoutTextDisplayCritical = "#a71b20";
export const ScoutTextDisplaySuccess = "#359220";
export const ScoutTextDisplayIncrease = "#359220";
export const ScoutTextDisplayDecrease = "#a71b20";
export const ScoutTextInteractivePrimary = "#029dd4";
export const ScoutTextInteractiveSecondary = "#4b5563";
export const ScoutTextInteractiveInfo = "#047eaf";
export const ScoutTextInteractiveWarning = "#ca8a04";
export const ScoutTextInteractiveCritical = "#a71b20";
export const ScoutTextInteractiveSuccess = "#359220";
export const ScoutTextInteractiveIncrease = "#359220";
export const ScoutTextInteractiveDecrease = "#a71b20";
export const ScoutTextInversePrimary = "#ffffff";
export const ScoutTextInverseSecondary = "#ffffffcc";
export const ScoutTypographyCaptionFontFamily =
  "Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif";
export const ScoutTypographyCaptionFontWeight = 400;
export const ScoutTypographyCaptionFontSize = "12px";
export const ScoutTypographyCaptionLineHeight = "18px";
export const ScoutTypographyCaptionLetterSpacing = "0";
export const ScoutTypographyLabelFontFamily =
  "Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif";
export const ScoutTypographyLabelFontWeight = 600;
export const ScoutTypographyLabelFontSize = "14px";
export const ScoutTypographyLabelLineHeight = "21px";
export const ScoutTypographyLabelLetterSpacing = "0";
export const ScoutTypographyLabelSmallFontFamily =
  "Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif"; // Compact label for condensed-size form fields. Same weight as label, one step down in size to match the condensed input rhythm.
export const ScoutTypographyLabelSmallFontWeight = 600; // Compact label for condensed-size form fields. Same weight as label, one step down in size to match the condensed input rhythm.
export const ScoutTypographyLabelSmallFontSize = "12px"; // Compact label for condensed-size form fields. Same weight as label, one step down in size to match the condensed input rhythm.
export const ScoutTypographyLabelSmallLineHeight = "18px"; // Compact label for condensed-size form fields. Same weight as label, one step down in size to match the condensed input rhythm.
export const ScoutTypographyLabelSmallLetterSpacing = "0"; // Compact label for condensed-size form fields. Same weight as label, one step down in size to match the condensed input rhythm.
export const ScoutTypographyBodySmallFontFamily =
  "Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif";
export const ScoutTypographyBodySmallFontWeight = 400;
export const ScoutTypographyBodySmallFontSize = "14px";
export const ScoutTypographyBodySmallLineHeight = "21px";
export const ScoutTypographyBodySmallLetterSpacing = "0";
export const ScoutTypographyBodyFontFamily =
  "Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif";
export const ScoutTypographyBodyFontWeight = 400;
export const ScoutTypographyBodyFontSize = "16px";
export const ScoutTypographyBodyLineHeight = "24px";
export const ScoutTypographyBodyLetterSpacing = "0";
export const ScoutTypographyBodyLargeFontFamily =
  "Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif";
export const ScoutTypographyBodyLargeFontWeight = 400;
export const ScoutTypographyBodyLargeFontSize = "20px";
export const ScoutTypographyBodyLargeLineHeight = "30px";
export const ScoutTypographyBodyLargeLetterSpacing = "0";
export const ScoutTypographyHeading3FontFamily =
  "Literata, Georgia, 'Times New Roman', serif";
export const ScoutTypographyHeading3FontWeight = 600;
export const ScoutTypographyHeading3FontSize = "20px";
export const ScoutTypographyHeading3LineHeight = "30px";
export const ScoutTypographyHeading3LetterSpacing = "0";
export const ScoutTypographyHeading2FontFamily =
  "Literata, Georgia, 'Times New Roman', serif";
export const ScoutTypographyHeading2FontWeight = 600;
export const ScoutTypographyHeading2FontSize = "24px";
export const ScoutTypographyHeading2LineHeight = "36px";
export const ScoutTypographyHeading2LetterSpacing = "0";
export const ScoutTypographyHeading1FontFamily =
  "Literata, Georgia, 'Times New Roman', serif";
export const ScoutTypographyHeading1FontWeight = 600;
export const ScoutTypographyHeading1FontSize = "32px";
export const ScoutTypographyHeading1LineHeight = "40px";
export const ScoutTypographyHeading1LetterSpacing = "0";
export const ScoutTypographyDisplaySmallFontFamily =
  "Literata, Georgia, 'Times New Roman', serif";
export const ScoutTypographyDisplaySmallFontWeight = 600;
export const ScoutTypographyDisplaySmallFontSize = "48px";
export const ScoutTypographyDisplaySmallLineHeight = "64px";
export const ScoutTypographyDisplaySmallLetterSpacing = "0";
export const ScoutTypographyDisplayLargeFontFamily =
  "Literata, Georgia, 'Times New Roman', serif";
export const ScoutTypographyDisplayLargeFontWeight = 600;
export const ScoutTypographyDisplayLargeFontSize = "64px";
export const ScoutTypographyDisplayLargeLineHeight = "76px";
export const ScoutTypographyDisplayLargeLetterSpacing = "0";
